cluster/src/main/java/com/hartwig/pipeline/Arguments.java

```diff
@@ -161,11 +161,11 @@ static String workingDir() {
                     .publishToTurquoise(false)
                     .pollInterval(DEFAULT_POLL_INTERVAL)
                     .refGenomeVersion(DEFAULT_REF_GENOME_VERSION)
-                    .maxConcurrentLanes(DEFAULT_MAX_CONCURRENT_LANES);
+                    .maxConcurrentLanes(DEFAULT_MAX_CONCURRENT_LANES)
+                    .imageName(EMPTY);
```

Comment:
As the argument is optional, we should use the features of Optional and not have another "empty" case of an empty string. You can either just not specify the image name in the builder or use Optional.empty(). I think the former is most consistent with what's already here.